src/Arguments/Parallel.php

```diff
@@ -18,7 +18,10 @@ final class Parallel implements ArgumentHandler
     private function editArguments(): void
     {
         $this->unsetArgument('--parallel')
-            ->unsetArgument('-p')
-            ->setArgument('--runner', Runner::class);
+            ->unsetArgument('-p');
```

Comment:
This here is the main change. Basically, if the user has defined a custom `--runner`, we respect that rather than forcing our own.